scripts/changelog/README.md

```diff
@@ -0,0 +1,68 @@
+# Auto-generate changelog for ksqlDB releases
+
+The ksqlDB repo uses conventional commits in order to assist with auto-generation of changelogs.
+
+## One-time setup
+
+Install the [tool](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-cli) we use to for generating changelogs:
+```
+npm install -g conventional-changelog-cli
+```
+
+## Generate changelog
+
+Assuming you've completed the one-time setup above, steps for generating a changelog are as follows.
+
+Suppose you're interested in generating a changelog of all changes in ksqlDB 0.8.0 that are new as of ksqlDB 0.7.0. Refer to these versions as `NEW_VERSION` and `OLD_VERSION`, respectively. The same process applies for other new/old ksqlDB version combinations as well (e.g., 0.8.1 and 0.8.0, or 0.8.0 and 0.7.1).
+
+These steps assume you're in the root of the `ksql` directory.
+
+### List commits on `NEW_VERSION`
+
+Start by listing all commits from the `NEW_VERSION`, which requires checking out the relevant branch/tag/commit. If the release tag exists:
+```
+git fetch --all --tags --prune
+git checkout tags/v0.8.0-ksqldb -b v0.8.0-ksqldb
```

Comment:
the branch name should not match the tag name, no?
```suggestion
git checkout tags/v0.8.0-ksqldb -b 0.8.0-ksqldb
```

Reply:
It shouldn't matter? @spena said he was having issues with ambiguous references yesterday but using a different branch name didn't resolve the issue.

Regardless, I've updated the PR to use @spena 's suggestion of checking out the reference without creating a branch.